core/jobs/batch_jobs/skill_migration_jobs_test.py

```diff
@@ -279,351 +275,12 @@ def test_skill_summary_of_unmigrated_skill_is_updated(self) -> None:
         self.assert_job_output_is([
             job_run_result.JobRunResult(stdout='SKILL PROCESSED SUCCESS: 1'),
             job_run_result.JobRunResult(stdout='SKILL MIGRATED SUCCESS: 1'),
-            job_run_result.JobRunResult(stdout='CACHE DELETION SUCCESS: 1')
         ])
 
         migrated_skill_summary_model = skill_models.SkillSummaryModel.get(
             self.SKILL_1_ID)
         self.assertEqual(migrated_skill_summary_model.version, 2)
 
-    def test_broken_cache_is_reported(self) -> None:
```

Comment:
@vojtechjelinek The `CountObjectsToJobRunResult` does not return a count for an empty PCollection, so it would be difficult to verify draining. Plus I think the behaviour is verified here 
https://github.com/oppia/oppia/blob/d7c0fb67e30cafa6d1d33d55128535ec36caadb5/core/jobs/batch_jobs/story_migration_jobs_test.py#L151

Reply:
Done